src/sentry/search/utils.py

```diff
@@ -326,7 +338,7 @@ def get_latest_release(
     if organization_id is None:
         project = projects[0]
         if hasattr(project, "organization_id"):
-            organization_id = project.organization_id
+            organization_id = project.organization_id  # type: ignore[union-attr]
```

Comment:
This seems like it should work, what was the error here?

Reply:
I think its cause `projects` is a `Sequence[Project | int]`, so the typing checker thinks its doing something like `<int>.organization_id`. This is covered by the `hasattr` check above, so I think this is safe.

Reply:
You could change the condition to be `isinstance(project, Project) and hasattr(project, "organization_id")`.

Then mypy wouldn't have the error.